src/grid_collisions.py

```diff
@@ -0,0 +1,60 @@
+from sys import (stdin, stdout)
+
+def valid_tile(y, x):
```

Comment:
we should have a `Map` class that stores it's data and we can call these function on. Otherwise we will pollute the. global namespace with too many variables.